### ISSUE TITLE
Fix heading structure in all pages except homepage

### ISSUE THREAD

**Issue by kburk1997** [YACS-RCOS/yacs-admin#197]
All pages except the homepage skip from h1 directly to h2. This is problematic for screen readers.
In addition, a lot of pages label what should be the h2 as h3 or h4. Change all pages to have a proper heading structure
